src/hooks/result.inc

```diff
@@ -7,6 +7,7 @@ execution_success() {
 
     if [ "$#" -eq 1 ]
     then
+        # shellcheck disable=SC2154
```

Comment:
Discovered when I was checking the syntax. It should be detected on #3, but it was still missing #5